lib/transmogrify.js

```diff
@@ -25,15 +25,19 @@ module.exports = function (url, options, cb) {
     var deferred = q.defer();
 
     // transmogrify all the things
-    if (url.substr(0, 4) != 'http') {
-      // assume its a file if it starts with http
+    if (typeof url == 'string' && url.substr(0, 4) == 'http') {
```

Comment:
<String> url war trimmed of whitespace before? otherwise substr(0,4) will not work ;)